browser-test/browser-test-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       - 10000:10000
 
   fake-idcs:
-    image: docker.io/civiform/oidc-provider:latest
+    image: civiform/oidc-provider
```

Comment:
Note to the reviewer: This follows the pattern we have for other images that we build locally in CI and want to use for the test run. Prior to this change, we built a local OIDC provider but then always pulled the version from DockerHub (and hence CI runs wouldn't reflect local changes).